python/kfserving/kfserving/__init__.py

```diff
@@ -39,6 +39,7 @@
 from .models.v1alpha2_inference_service_spec import V1alpha2InferenceServiceSpec
 from .models.v1alpha2_inference_service_status import V1alpha2InferenceServiceStatus
 from .models.v1alpha2_endpoint_spec import V1alpha2EndpointSpec
+from .models.v1alpha2_onnx_spec import V1alpha2ONNXSpec
```

Comment:
nit: it should go down with the other framework specs on like 46

Reply:
Done